.gitignore

```diff
@@ -8,6 +8,8 @@ src/FridayCore.AccountResetRules/bin/
 src/FridayCore.AccountResetRules/obj/
 src/FridayCore.AutoPackages/bin/
 src/FridayCore.AutoPackages/obj/
+src/FridayCore.ApplicationContainer/**/bin/
+src/FridayCore.ApplicationContainer/**/obj/
```

Comment:
`/**/bin/` looks dangerous, it is possible that somebody adds files in the following dir and these files will be lost:
src/FridayCore.ApplicationContainer/Engine/Obj/Obj.cs